src/containers/NewPlayerForm/NewPlayerForm.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
This could probably either be done with a utility higher order component or moved to a component. It should **definitely not** render UI if it's in a container.